### ISSUE TITLE
Cannot start videos after last update

### ISSUE THREAD

**Issue by meme1337** [croneter/PlexKodiConnect#1493]
## Help yourself
* I did try to restart Kodi :-)
* I checked the [PKC Frequently Asked Questions on the PKC wiki](https://github.com/croneter/PlexKodiConnect/wiki/faq)
* I did try to reset the Kodi database by going to `PKC Settings -> Advanced -> "Reset the database and optionally reset PlexKodiConnect"` and then hitting YES, NO
* I did check the [existing issues on Github](https://github.com/croneter/PlexKodiConnect/issues)
* I  did also uninstall and reinstall PKC ;)

## Describe the bug
When I try to start a video i get the following error:
`create_listitem() got an unexpected keyword argument 'listitem'`

## To Reproduce
1. Install latest PKC
2. Update the DB if necessary
3. Start any video (TV show or Movie)

## Expected behavior
Everything was working before latest update.

## You need to attach a KODI LOG FILE!
```
2021-05-24 19:47:25.017 T:31882   ERROR <general>: Playlist Player: skipping unplayable item: 0, path [plugin://plugin.video.plexkodiconnect.tvshows/14388/?mode=play&plex_id=14398&plex_type=episode&filename=Chuck - S01E09.mkv]
2021-05-24 19:47:25.018 T:31882   DEBUG <general>: Playlist Player: no more playable items... aborting playback
2021-05-24 19:47:25.020 T:32140   DEBUG <general>: Thread BackgroundLoader start, auto delete: false
2021-05-24 19:47:25.035 T:32140   DEBUG <general>: Thread BackgroundLoader 161982944496 terminating
2021-05-24 19:47:25.251 T:32024   DEBUG <general>: PLEX.transfer: Received
2021-05-24 19:47:25.251 T:32024   DEBUG <general>: PLEX.playback: Using resume False
2021-05-24 19:47:25.251 T:32024   DEBUG <general>: PLEX.playback: Resuming: False. Playing trailers: False
2021-05-24 19:47:25.252 T:32024   DEBUG <general>: PLEX.playlist_func: Playlist cleared: {'playlistid': 1, 'id': None, 'version': None, 'type': 'video', 'selectedItemID': None, 'selectedItemOffset': None, 'shuffled': 0, 'repeat': 0, 'kodi_playlist_playback': False, 'pkc_edit': False, 'items': []}
2021-05-24 19:47:25.254 T:32127   DEBUG <general>: .
2021-05-24 19:47:25.453 T:32034   DEBUG <CAddonSettings[plugin.video.plexkodiconnect]>: Skipped 1 duplicate messages..
2021-05-24 19:47:25.453 T:32034   DEBUG <CAddonSettings[plugin.video.plexkodiconnect]>: trying to load setting definitions from old format...
2021-05-24 19:47:25.651 T:32127 WARNING <general>: Skipped 1 duplicate messages..
2021-05-24 19:47:25.651 T:32127 WARNING <general>: CPythonInvoker(8, /storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.plexkodiconnect.tvshows/default.py): the python script "/storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.plexkodiconnect.tvshows/default.py" has left several classes in memory that we couldn't clean up. The classes include: N9XBMCAddon9xbmcaddon5AddonE
2021-05-24 19:47:25.652 T:32127    INFO <general>: Python interpreter stopped
2021-05-24 19:47:25.652 T:32127   DEBUG <general>: Thread LanguageInvoker 162020844784 terminating
2021-05-24 19:47:25.652 T:31919   DEBUG <general>: PLEX.kodimonitor: Method: Playlist.OnClear Data: {'playlistid': 1}
2021-05-24 19:47:25.671 T:32034   DEBUG <CAddonSettings[plugin.video.plexkodiconnect]>: trying to load setting definitions from old format...
2021-05-24 19:47:25.686 T:32105   DEBUG <general>: PLEX.metadata_movies: Found a new trailer for movie The Rock: plugin://plugin.video.youtube/?action=play_video&videoid=bI6Mbg_bMzQ
2021-05-24 19:47:25.686 T:32034   DEBUG <CAddonSettings[plugin.video.plexkodiconnect]>: trying to load setting definitions from old format...
2021-05-24 19:47:25.715 T:32105   DEBUG <general>: PLEX.metadata_movies: Processing trailer for movie 11802
2021-05-24 19:47:25.737 T:32024   DEBUG <general>: PLEX.playlist_func: Updated playlist from xml: {'playlistid': 1, 'id': 4066, 'version': 1, 'type': 'video', 'selectedItemID': 51370, 'selectedItemOffset': 0, 'shuffled': 0, 'repeat': 0, 'kodi_playlist_playback': False, 'pkc_edit': False, 'items': []}
2021-05-24 19:47:25.756 T:32105   DEBUG <CAddonSettings[metadata.themoviedb.org.python]>: trying to load setting definitions from old format...
2021-05-24 19:47:25.764 T:32024   ERROR <general>: PLEX.utils: Error encountered:  - create_listitem() got an unexpected keyword argument 'listitem'
2021-05-24 19:47:25.765 T:32105   DEBUG <general>: using movie id of 4552 to get movie details
2021-05-24 19:47:25.765 T:32105   DEBUG <general>: Calling URL "https://api.themoviedb.org/3/movie/4552?api_key=f090bb54758cabf231fb605d3e3e0468&language=en-US&append_to_response=trailers%2Cimages%2Creleases%2Ccasts%2Ckeywords"
2021-05-24 19:47:25.801 T:31919   DEBUG <general>: PLEX.kodimonitor: Detected PKC clear - ignoring
2021-05-24 19:47:25.801 T:31919   DEBUG <general>: PLEX.kodimonitor: PKC settings change detected
2021-05-24 19:47:25.804 T:32024   ERROR <general>: PLEX.utils: _____________________________________________________________
2021-05-24 19:47:25.804 T:32024   ERROR <general>: PLEX.utils:     Traceback (most recent call last):
2021-05-24 19:47:25.804 T:32024   ERROR <general>: PLEX.utils:       File "/storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.plexkodiconnect/resources/lib/backgroundthread.py", line 373, in _runTask
2021-05-24 19:47:25.805 T:32024   ERROR <general>: PLEX.utils:         task._run()
2021-05-24 19:47:25.805 T:32024   ERROR <general>: PLEX.utils:       File "/storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.plexkodiconnect/resources/lib/backgroundthread.py", line 307, in _run
2021-05-24 19:47:25.805 T:32024   ERROR <general>: PLEX.utils:         self.run()
2021-05-24 19:47:25.806 T:32024   ERROR <general>: PLEX.utils:       File "/storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.plexkodiconnect/resources/lib/playback_starter.py", line 43, in run
2021-05-24 19:47:25.806 T:32024   ERROR <general>: PLEX.utils:         playback.playback_triage(plex_id=params.get('plex_id'),
2021-05-24 19:47:25.806 T:32024   ERROR <general>: PLEX.utils:       File "/storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.plexkodiconnect/resources/lib/playback.py", line 46, in playback_triage
2021-05-24 19:47:25.806 T:32024   ERROR <general>: PLEX.utils:         _playback_triage(plex_id, plex_type, path, resolve, resume)
2021-05-24 19:47:25.807 T:32024   ERROR <general>: PLEX.utils:       File "/storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.plexkodiconnect/resources/lib/playback.py", line 131, in _playback_triage
2021-05-24 19:47:25.807 T:32024   ERROR <general>: PLEX.utils:         _playback_init(plex_id, plex_type, playqueue, pos, resume)
2021-05-24 19:47:25.807 T:32024   ERROR <general>: PLEX.utils:       File "/storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.plexkodiconnect/resources/lib/playback.py", line 244, in _playback_init
2021-05-24 19:47:25.807 T:32024   ERROR <general>: PLEX.utils:         stack = _prep_playlist_stack(xml, resume)
2021-05-24 19:47:25.807 T:32024   ERROR <general>: PLEX.utils:       File "/storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.plexkodiconnect/resources/lib/playback.py", line 361, in _prep_playlist_stack
2021-05-24 19:47:25.808 T:32024   ERROR <general>: PLEX.utils:         listitem = api.listitem()
2021-05-24 19:47:25.808 T:32024   ERROR <general>: PLEX.utils:       File "/storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.plexkodiconnect/resources/lib/plex_api/base.py", line 646, in listitem
2021-05-24 19:47:25.808 T:32024   ERROR <general>: PLEX.utils:         return widgets.create_listitem(item, as_tuple=False, listitem=listitem)
2021-05-24 19:47:25.808 T:32024   ERROR <general>: PLEX.utils:     TypeError: create_listitem() got an unexpected keyword argument 'listitem'
2021-05-24 19:47:25.808 T:32024   ERROR <general>: PLEX.utils: _____________________________________________________________
```

**Comment by croneter** [comment]
Outch, thanks for the report!

**Comment by meme1337** [comment]
> Outch, thanks for the report!

No problem mate! Let me know if I can do anything to troubleshoot the issue. 

I have to add that I’m using latest Kodi Matrix on an nVidia Shield 2017.

**Comment by croneter** [comment]
Fixed in beta 3.2.2
